test/studies/champs/sub_test

```diff
@@ -43,53 +67,64 @@ if [ "${CHPL_LLVM}" != "none" ] ; then
   apply_patch $CHAMPS_PATCH_PATH/optional/llvm.patch
 fi
 
-# copy the prepartitoned grid for 16 locales into scratch space for better IO
-# performance
-cp $CHAMPS_DATA_PATH/CRMHL_coarse_1152B.cgns $CHAMPS_HOME/
-
-# in the system where we run this, compute nodes don't have access to our shared
-# non-scratch space. I do not want to store those in the scratch space for good.
-# So, what we are doing here is to copy the stuff from CHAMPS_CFG_PATH and
-# CHAMPS_DEP_PATH to the scratch space for running the code.
-cp $CHAMPS_CFG_PATH/flow.in $CHAMPS_HOME/
-
-# ... and this is where it really gets unfortunate.
-cp -r $CHAMPS_DEP_PATH $CHAMPS_HOME/
-export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$CHAMPS_HOME/deps-manual/lib
+if [ ! -z "$CHAMPS_QUICKSTART" ]; then
+  echo "[Patch commit created]"
```

Comment:
(assuming you didn't do it somewhere above) Maybe at this point you'll want to `git checkout -b patched` or something like that just so it's on its own named branch instead of being committed to main (or whatever).

Reply:
https://github.com/chapel-lang/chapel/pull/23444/commits/81dd3106bc8edbb003e8b7b78c89c7961ab0e0c2